src/ghostfat.c

```diff
@@ -319,16 +318,16 @@ void uf2_read_block (uint32_t block_no, uint8_t *data)
   else if ( block_no < BPB_TOTAL_SECTORS )
   {
     sectionIdx -= FS_START_CLUSTERS_SECTOR;
-    if ( sectionIdx < NUM_FILES - 1 )
+    if ( sectionIdx < (NUM_FILES - 1) * BPB_SECTORS_PER_CLUSTER )
```

Comment:
<details><summary>Comments to help refresh my memory</summary><P/>

* `sectionIdx` is renamed to `sectionRelativeSector`
* The root directory in FAT16 is NOT part of the data volume
* Therefore, if only having files in root as ghostfat does, the first data sectors are for the file contents.

</details>

Yes, good!

Please update the static assertion to reflect that each file in ghostfat must fit within a single cluster, not a single sector (e.g., ~lines 142 here)